### DISCUSSION TITLE
key repeat is intermittent on tulip hardware

### DISCUSSION THREAD

**Discussion by drepetto** [shorepine/tulipcc#83] [General]
Key repeat sometimes works in terminal, almost never works in edit.py. When it does work it's just for a bit then it konks out.  I'm using an old Dell USB keyboard, no hub or anything fancy about it.

**Comment by bwhitman** [comment]
I cannot replicate this at all. I'm guessing for now this is something about your hardware. Let's put it in discussions for other people that may be dealing with it.

**Reply by drepetto** [reply]
I think @dpwe said he has also had this problem?

**Comment by drepetto** [comment]
related to: https://github.com/bwhitman/tulipcc/issues/77#issue-1710226200

??

**Reply by bwhitman** [reply]
Possibly. But it's not happening ever for me, and I know there's other issues on your board

**Reply by dpwe** [reply]
I think you're holding in wrong.

**Comment by bwhitman** [comment]
I've made some changes to make keyboard handling better. but it looks like a bodge is necessary to really fix it, and i'll try to update this for a new board rev.

**Reply by bwhitman** [reply]
See #94
